### PR TITLE
Added changes related venu announcement blocks and menu.

### PR DESCRIPTION
**Pull request by Rangaswini**

### CODE REVIEW

**Review by coderkakarrot — APPROVED** [review]
LGTM

File: docroot/themes/custom/dcp/sass/components/_venue-block.scss [hunk 1]
```diff
@@ -38,14 +38,15 @@
     background-image: linear-gradient($light-grey, white);
     justify-content: space-between;
 
-    @include breakpoint(phablet) {
+    @include breakpoint(tablet) {
       display: block;
       margin: 0;
       padding-right: 0;
       padding-bottom: 50px;
+      // padding: 15px 0 50px 0;
```
Comment (prashant114606-zz) [inline docroot/themes/custom/dcp/sass/components/_venue-block.scss]: please remove commented code.

**Comment by prashant114606-zz** [comment]
LGTM

File: docroot/themes/custom/dcp/sass/components/_venue-block.scss (hunk 1 above)
Comment (Rangaswini) [reply docroot/themes/custom/dcp/sass/components/_venue-block.scss]: Removed commented code

**Pull request merged**